shopfloor/services/zone_picking.py

```diff
@@ -4,12 +4,14 @@
 import functools
 from collections import defaultdict
 
+from odoo.exceptions import UserError
```

Comment:
Should be dropped now as replaced by ConcurentWorkOnTransfer

Reply:
Done